src/server/utilities/cspHeader/index.js

```diff
@@ -58,8 +54,8 @@ const advertisingDirectives = {
     'https://collector.effectivemeasure.net',
     'https://me-ssl.effectivemeasure.net',
     'https://pixel.adsafeprotected.com',
-    'https://sb.scorecardresearch.com',
     'https://static.adsafeprotected.com',
+    'https://sb.scorecardresearch.com',
```

Comment:
Did you intend to remove this or just move it?

Reply:
Just move it so the adsafeprotected domains are "together"

Reply:
Thanks @bbc-fatima, approved the PR 🙂